website/app/styles/pages/application/banner.scss

```
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// BANNER
+
+@use "../../breakpoints" as breakpoint;
+
+$horizontal-padding: 90px; // reserved space for the "close" button
+
+.doc-page-banner {
+  position: relative;
+  z-index: 2;
+  display: flex;
+  grid-area: banner;
+  gap: 6px;
+  align-items: center;
+  justify-content: center;
+  padding: 8px $horizontal-padding;
+  color: #fff;
+  background: linear-gradient(135deg, #4f53d0 0%, #7545cb 100%);
+
+  body:not(.index) & {
+    @include breakpoint.with-fixed-sidebar() {
+      padding-left: calc(var(--doc-page-sidebar-width) + $horizontal-padding); // we need to compensate for the fixed position of the sidebar
+    }
+  }
+}
+
+.doc-page-banner__icon {
+  flex: none;
+}
+
+.doc-page-banner__text {
+  margin: 0;
+
+  a {
+    color: inherit;
+  }
+}
+
+.doc-page-banner__close {
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  color: #fff;
+  background-color: transparent;
+  border: 1px solid transparent;
```

Adding a visible border to make it appear more button-like.

```suggestion
  border: 1px solid #fff;
  border-radius: 3px;
  padding: 2px;
```